src/PubmedId.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Altmetric\Identifiers;
+
+class PubmedId
+{
+    public static function extract($str)
+    {
+        preg_match_all('/(?<=^|\s)0*(?!0)(\d+)(?=$|\s)/', $str, $matches);
+
+        return $matches[0];
```

Comment:
As `$matches[0]` contains the full matches, does this mean that we don't use the capturing groups in the pattern?